bindings/qt/QTMegaGlobalListener.cpp

```diff
@@ -43,6 +43,22 @@ void QTMegaGlobalListener::onReloadNeeded(MegaApi *api)
     QCoreApplication::postEvent(this, event, INT_MIN);
 }
 
+void QTMegaGlobalListener::onEvent(MegaApi *api, MegaEvent *e)
+{
+    switch(e->getType())
+    {
+        case MegaEvent::EVENT_ACCOUNT_CONFIRMATION:
+        {
+            QTMegaEvent *event = new QTMegaEvent(api, (QEvent::Type)QTMegaEvent::OnEvent);
+            event->setEvent(e ? e->copy() : NULL);
```

Comment:
This doesn't make sense because if `e` is NULL the code would crash some lines above.